src/storage/txn/task.rs

```diff
@@ -58,25 +71,12 @@ impl Task {
         self.extra_op
     }
 
+    /// Set_extra_op is called after getting snapshot and before the command is
```

Comment:
```suggestion
    /// set_extra_op is called after getting snapshot and before the command is
```
Btw we don't need to start the comment with the function's name, which is the convention of golang.